### ISSUE TITLE
glTF import crashes if `physics/3d/run_on_separate_thread` is enabled

### ISSUE THREAD

**Issue by SmartySmart702** [godotengine/godot#61250]
### Godot version

4.0 alpha 8

### System information

Win 10

### Issue description

If physics/3d/run_on_separate_thread is true, drag and drop an imported glb scene into another scene causes a crash.

### Steps to reproduce

1) Create a new test project and open it.
2) Create a test glb and save it in your project folder (in my case Blender default cube).
3) Create a new Scene, add a Spatial as root and save the scene.
4) Select root and drag and drop the glb filt into your scene.
Everything works fine.
5) Enable "physics/3d/run_on_separate_thread".
6) Save and restart Godot.
7) Select root of your scene and drag and drop the glb filt into your scene.
Crash.

**Comment by Calinou** [comment]
@SmartySmart702 Can you reproduce this on 3.4.4?

**Comment by SmartySmart702** [comment]
> @SmartySmart702 Can you reproduce this on 3.4.4?
I dont know how to properly test this in 3.4.4. I have enabled physics/3d/run_on_separate_thread and I could not reproduce the crash but I'm not even sure if this property exists in 3.4.4.

**Comment by fire** [comment]
I'm interested in looking at this.

**Comment by fire** [comment]
Can you provide a sample project @SmartySmart702 so we can work on the same problem?

**Comment by Imvoo** [comment]
I was able to replicate this crash in my own project and in a brand new project with Blender's default cube (as suggested).

- I used v4.0.beta2.mono.official [f8745f2f7].
- I set `physics/3d/run_on_separate_thread` to `true`.
  - (`Project -> Project Settings -> Physics -> 3D -> Run On Separate Thread (check this box)`)

I was not able to replicate this on Godot 3.x as the `run_on_separate_thread` option is not available for 3D in Godot 3.x.

Minimal reproduction project: [test-physics-import-crash.zip](https://github.com/godotengine/godot/files/9713240/test-physics-import-crash.zip)

Dragging `cube.glb` from the `FileSystem` within Godot, and dropping it into the scene will force a crash.
